cubed/core/ops.py

```diff
@@ -109,7 +109,7 @@ def from_zarr(store, path=None, spec=None) -> "Array":
         The array loaded from Zarr storage.
     """
     name = gensym()
-    target = zarr.open(store, path=path, mode="r")
+    target = zarr.open(store, path=path, mode="r", storage_options=spec.storage_options)
```

Comment:
This will fail if `spec` is `None`, which it can be when the default config is being used. You can fix this by adding a line like this:

https://github.com/cubed-dev/cubed/blob/3d08513359ef168979a372a702d2f41f79e6f769/cubed/core/array.py#L47

Reply:
Done. Sorry for the confussion.